homeassistant/components/tado/climate.py

```diff
@@ -328,7 +347,10 @@ def __init__(
         self._current_tado_vertical_swing = TADO_SWING_OFF
         self._current_tado_horizontal_swing = TADO_SWING_OFF
 
-        capabilities = tado.get_capabilities(zone_id)
+        if tado.is_x:
+            capabilities = {"type": TYPE_HEATING}
+        else:
+            capabilities = tado.get_capabilities(zone_id)
```

Comment:
since this is duplicate code, maybe wrap this in a helper?

Reply:
good idea, I've put it in the `tado.get_capabilities` function